Sources/TuistCore/Models/TargetAction.swift

```diff
@@ -92,7 +165,9 @@ public struct TargetAction: Equatable {
     /// - Returns: Shell script that should be used in the target build phase.
     /// - Throws: An error if the tool absolute path cannot be obtained.
     public func shellScript(sourceRootPath: AbsolutePath) throws -> String {
-        if let path = path {
+        if case let Script.text(text) = script {
+            return text
```

Comment:
I think we want to run `spm_chomp().spm_chuzzle`, too? Also refactoring the if lets to a switch case on the enum `Script` would be beneficial here

Reply:
Good call!